subgraphs/euler-finance/src/mappings/handlers.ts

```diff
@@ -293,3 +310,132 @@ export function handleGovSetReserveFee(event: GovSetReserveFee): void {
     event,
   );
 }
+
+export function handleStake(event: Stake): void {
```

Comment:
tbh, I am kinda struggling to follow all of the moving parts in this function.. Could you also include notes about the function to find rewards that you linked in the PR?